app/src/main/java/org/gdg/frisbee/android/chapter/NewsFragment.java

```diff
@@ -55,8 +56,13 @@ public static NewsFragment newInstance(String plusId) {
     public void onResume() {
         super.onResume();
 
-        for (int i = 0; i <= getListView().getChildCount(); i++) {
-            mAdapter.updatePlusOne(getListView().getChildAt(i));
+        RecyclerView recyclerView = getListView();
+        for (int i = 0; i <= recyclerView.getChildCount(); i++) {
```

Comment:
Should we cache the size inside the for loop? 

`for (int i = 0, size = recyclerView.getChildCount(); i <= size; i++)`